packaging/msi/msi-builder/build-launcher.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
This script is a rename of `packaging/msi/msi-builder/docker-entrypoint.sh` plus the addition of the parameters `REPO_DIR` and `WORK_DIR`.